profile/README.md

```diff
@@ -4,9 +4,55 @@
 packages to be uploaded, so why not upload Godot addons there to be automatically
 managed?
 
+## Using Packages Quickstart
+
+1. Find some [Godot packages on NPM](https://www.npmjs.com/search?q=keywords:godot-engine)
+2. Add them to the `godot.package` file [(See the `godot.package` section)](#godotpackage)
+3. Install your [client](#clients) of choice
+3. Run your clients update function
+
+## `godot.package`
+
+Describes packages to be installed. This _should_ be modified
+by the user.
+
+This file can be in different markup languages:
+
+<details open>
+<summary><b>JSON</b></summary>
+
+```jsonc
+{
+  // comments are supported
+  "packages": {
+    "package": "1.0.0", // as are trailing commas
```

Comment:
```suggestion
    "my_package": "1.0.0", // as are trailing commas
```
Rename `package` to `my_package` so that the name doesn't blend together with `packages`.

Reply:
Good idea